Submitted By: BLFS Book <blfs-book@linuxfromscratch.org>
Date: 2003-10-05
Initial Package Version: 6.1
Origin: LFS Book
Description: Uses FHS compliant locations for vimrc and gvimrc.
diff -ur vim61.orig/src/feature.h vim61/src/feature.h
--- vim61.orig/src/feature.h	2002-03-09 09:17:30.000000000 -0600
+++ vim61/src/feature.h	2003-02-03 21:42:41.000000000 -0600
@@ -826,12 +826,12 @@
 /*
  * SYS_VIMRC_FILE	Name of the system-wide .vimrc file.
  */
-/* #define SYS_VIMRC_FILE	"/etc/vimrc" */
+#define SYS_VIMRC_FILE	"/etc/vimrc"
 
 /*
  * SYS_GVIMRC_FILE	Name of the system-wide .gvimrc file.
  */
-/* #define SYS_GVIMRC_FILE	"/etc/gvimrc" */
+#define SYS_GVIMRC_FILE	"/etc/gvimrc"
 
 /*
  * DFLT_HELPFILE	Name of the help file.
